maps/mp/isnipe/settings.gsc

```diff
@@ -26,7 +26,6 @@ apply()     //TODO: camelCase everywhere, fix this list up. many things no longe
     PlayerMaxHealth = 20; //Settings this to more than 20 could possible prevent 1 shotting people
     EnableKillcam = true;
     ShowPlayersAlive = true;
-    ShowTopPlayer = true;
```

Comment:
The settings have been removed from settings.gsc. We should probably add a settings module.